packages/design/src/colors.css

```diff
@@ -110,8 +112,8 @@
   --color-grey--lightest: rgb(244, 244, 244);
   --color-grey--dark: rgb(118, 118, 118);
 
-  --color-greyBlue--rgb: 101, 120, 132;
-  --color-greyBlue: rgb(var(--color-greyBlue--rgb));
+  --color--greyBlue--rgb: 101, 120, 132;
```

Comment:
Hmm this might be broken, I'm pretty sure this is a typo that I introduced and then later fixed. Just want to check this out reallll quick